lib/workers/global/index.ts

```diff
@@ -103,6 +104,7 @@ export async function resolveGlobalExtends(
 
 export async function start(): Promise<number> {
   let config: AllConfig;
+  let sshSocket: SshSocket = undefined;
```

Comment:
```suggestion
  let sshSocket: SshSocket | undefined = undefined;
```
better use `null` instead of `undefined`

Reply:
Done